### PR TITLE
Added README and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by TSAnsari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# framegrabber
+FRAMEGRABBER
+============
+A Yocto/OpenEmbedded layer providing BSP for Automotive Reference Platform.
```
Comment (TSAnsari) [inline README.md]: Sorry missed changing this. Will fix.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+How to contribute
+=================
+We are happy to accept contributions. To accept a contribution we require:
+
+* A developer certificate of origin: see http://developercertificate.org/
```
Comment (frznlogic) [inline CONTRIBUTING.md]: Where did the template for this contributing.md file come from? Do we really use DC?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    - A "signed-off-by" line in the commits is interpreted as a DCO from our perspective
```
Comment (frznlogic) [inline CONTRIBUTING.md]: Dco? Should have more description imho

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
-# framegrabber
+FRAMEGRABBER
+============
+Driver module to load video frames captured from ARP cameras in to memory.
+
+Maintained at https://github.com/pelagicore/framegrabber
+
```
Comment (frznlogic) [inline README.md:6]: This should be in a separate file called LICENSING or something similar imho?

**Review by frznlogic — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (jeremiah) [reply CONTRIBUTING.md]: We don't always use the DCO, but when we do it saves our bacon. The template is from somewhere, but most of our projects have this or similar: https://github.com/Pelagicore/softwarecontainer/blob/master/CONTRIBUTING.md

**Review by jeremiah — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (jeremiah) [reply README.md:6]: Perhaps, but usually you have a license in LICENSE or COPYING and you explain contribution policy in CONTRIBUTING and/or README. But naturally this is not set in stone.

**Review by jeremiah — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (frznlogic) [reply CONTRIBUTING.md]: In this instance I was not referring to the dco, I was on the phone and tried to make an acronym of developer certificate.org.

Anyways, apparently we do, thanks for enlightening me 🙂.

File: README.md (hunk 4 above)
Comment (frznlogic) [reply README.md:6]: Exactly, the license should be split into a separate file was my point. I forgot the exact name of it when I was commenting before.

File: CONTRIBUTING.md (hunk 2 above)
Comment (TSAnsari) [reply CONTRIBUTING.md]: This was copied from the meta-pelux's contributing.md file. But perhaps I should follow the templates in softwarecontainer as described by Jeremiah

File: README.md (hunk 4 above)
Comment (TSAnsari) [reply README.md:6]: The LICENSE file has already been delivered by Jeremiah in another commit.

File: CONTRIBUTING.md (hunk 2 above)
Comment (frznlogic) [reply CONTRIBUTING.md]: No problem imho, if they are the same, they are the same.

**Review by frznlogic — APPROVED** [review]

**Pull request merged**